data/fields/fuel/fuel_multi.json

```diff
@@ -0,0 +1,51 @@
+{
+    "key": "fuel:",
+    "type": "multiCombo",
+    "label": "Fuel Types",
+    "strings": {
+        "options": {
+            "diesel": "Diesel",
+            "GTL_diesel": "GTL Diesel",
+            "HGV_diesel": "HGV Diesel",
```

Comment:
“HGV” is a British acronym completely unknown in American English, where this type would be called “Truck Diesel”. (It’s an odd value. Apparently it refers to the kind of nozzle / what vehicle class can use the pump, not the type of fuel supplied.)